rest_docs/rest-catalog-open-api-v0.1.yaml

```diff
@@ -0,0 +1,657 @@
+                $ref: '#/components/schemas/TableAlreadyExistsError'
+              example: {
+                "error": {
+                  "message": "Table already exists",
+                  "type": "AlreadyExistsException",
+                  "code": 409
+                } }
+
+components:
+
+  ##############################
+  # Application Schema Objects #
+  ##############################
+  schemas:
+    ResponseDataObject:
+      type: object
+      description: JSON data payload returned in a successful response body
+      properties:
+        data:
+          type: object
+          description: Wrapper for the response of a successful request
+      example: { "data": { "identifiers": [ "office.employees", "office.dogs", "office.cats" ] } }
+    ResponseErrorObject:
+      type: object
+      description: JSON error payload returned in a response with further details on the error
+      required:
+        - message
+        - type
+        - code
+      properties:
+        message:
+          type: string
+          description: Human-readable error message
+        type:
+          type: string
+          description: Internal type of the error, such as an exception class
+          example: NoSuchNamespaceException
+        code:
+          type: integer
+          minimum: 100
+          maximum: 600
+          description: HTTP response code
+          example: 404
+    TableIdentifier:
+      type: object
+      required:
+        - namespace
+      properties:
+        namespace:
+          type: array
+          description: Individual levels of the namespace
+          items:
+            type: string
+          nullable: false
+        name:
+          type: string
+          nullable: false
+    CreateNamespaceRequest:
+      type: object
+      required:
+        - namespace
+      properties:
+        namespace:
+          type: array
+          description: Individual levels of the namespace
+          items:
+            type: string
+        properties:
+          type: object
+          description: Configured properties for the namespace
+          example: '{ "owner": "Hank Bendickson" }'
+    RenameTableRequest:
+      type: object
+      properties:
+        sourceTableIdentifier:
+          $ref: '#/components/schemas/TableIdentifier'
+        destinationTableIdentifier:
+          $ref: '#/components/schemas/TableIdentifier'
+    UpdatePropertiesRequest:
+      type: object
+      properties:
+        toRemove:
+          type: array
+          uniqueItems: true
+          items:
+            type: string
+          example: '[ "department", "access_group" ]'
+        toUpdate:
+          uniqueItems: true
+          type: object
+          items:
+            type: string
+          example: { "owner": "Hank Bendickson" }
+    UpdatePropertiesResponse:
+      type: object
+      description: JSON data response for a synchronous update properties request.
+      properties:
+        updated:
```

Comment:
Wouldn't it be easier to just return the full set of properties instead of splitting it up into updated, removed and a maybe not present `notPresent`?